openjdk/src/test/java/org/conscrypt/AbstractSessionContextTest.java

```diff
@@ -16,16 +16,25 @@
 
 package org.conscrypt;
 
+import com.sun.org.apache.xerces.internal.impl.dv.util.Base64;
```

Comment:
I don't think this will work when compiled into CTS. Doesn't look like it's really needed at any rate.